app/lib/matador/components/errors/error.tsx

```diff
@@ -6,14 +6,14 @@ import {
   Text,
 } from "@mantine/core";
 import type { ThrownResponse } from "@remix-run/react";
-import { useEffect, useState } from "react";
+import { useState, useEffect } from "react";
 import { BiMessageError } from "react-icons/bi";
-import { themeKeyLocalStorage } from "~/lib/matador/constants";
-import { Navigations } from "~/lib/matador/helpers/ui-helpers";
-import { NavBar } from "./navbar";
+import { themeKeyLocalStorage } from "../../constants";
+import { Navigations } from "../../helpers/ui-helpers";
+import { NavBar } from "../navbar";
```

Comment:
Why this change? I think it is better to use the `~` notation

Reply:
Fixed with 104ea5d1507c64cbae3a06d987ebd3bf1526e989